tgui/packages/common/string.js

```diff
@@ -175,7 +175,7 @@ export const decodeHtmlEntities = (str) => {
     // Basic entities
     .replace(translate_re, (match, entity) => translate[entity])
     // Decimal entities
-    .replace(/&#?([0-9]+);/gi, (match, numStr) => {
+    .replace(/&#?([0-9]+);?/gi, (match, numStr) => {
```

Comment:
this was an issue: decodeHtmlEntities was not registering "&#39", which is the apostrophe